lightwood/mixers/helpers/default_net.py

```diff
@@ -74,7 +75,7 @@ def __init__(self, ds, dynamic_parameters):
 
         layers = []
         for ind in range(len(shape) - 1):
-            layers.append(torch.nn.Linear(shape[ind],shape[ind+1]))
+            layers.append(PLinear(shape[ind],shape[ind+1]))
```

Comment:
We should not remove the Linear layer completely, instead, we should add a flag to toggle the linear layer on/off or mass `layer_type` to the default network when we initialize it (I think the former is better).

If we decide that the PLinear layer is better we can make it a default later.

Reply:
agree